gnomad/utils/constraint.py

```diff
@@ -808,14 +808,20 @@ def get_constraint_grouping_expr(
     # Add 'transcript' and 'canonical' annotation if requested.
     if include_transcript_group:
         groupings["transcript"] = vep_annotation_expr.transcript_id
-    if include_canonical_group:
+    if preferred_transcript_group == "canonical":
         groupings["canonical"] = hl.or_else(vep_annotation_expr.canonical == 1, False)
+    elif preferred_transcript_group == "mane":
+        groupings["mane"] = hl.or_else(
+            hl.is_defined(vep_annotation_expr.mane_select), False
+        )
 
     return groupings
 
 
 def annotate_exploded_vep_for_constraint_groupings(
-    ht: hl.Table, vep_annotation: str = "transcript_consequences"
+    ht: hl.Table,
+    vep_annotation: str = "transcript_consequences",
+    preferred_transcript_group: str = "canonical",
```

Comment:
Maybe have `include_canonical_group=True` and `include_mane_group=False`, and say that at least one must be set if `vep_annotation == "transcript_consequences"` and they will be ignored when `vep_annotation == "worst_csq_by_gene"`